### PR TITLE
[10.0][ADD] Group "Edit Favicon" in web_favicon

### PR DESCRIPTION
**Pull request by tarteo**
This addition adds a group "Edit Favicon" which allows a user (if member of) to edit the favicon of companies.

I could've also used base.group_system, but I thought creating a new group would be nicer.

### CODE REVIEW

**Comment by dreispt** [comment]
Why a new group just for this?
I would rather (re)use the Configuration grosup `base.group_system`.

**Comment by tarteo** [comment]
@dreispt To make it more flexible.

**Comment by tarteo** [comment]
@dreispt But I've no problem with changing it to base.group_system as I mentioned in the description of this PR. What do you think?

**Comment by dreispt** [comment]
IMO it's not worth a new group, but let's see what others think.
BTW, +1 for the feature 😄

File: web_favicon/security/web_favicon_security.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="group_edit_favicon" model="res.groups">
```
Comment (felixvillafranca) [inline web_favicon/security/web_favicon_security.xml]: I agree with @dreispt, remove the group please and use the one provided by Odoo.

**Review by felixvillafranca — CHANGES_REQUESTED** [review]
The idea is good, but adding a grup just for this makes no sense.

File: web_favicon/views/res_company.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
         <field name="inherit_id" ref="base.view_company_form" />
         <field name="arch" type="xml">
             <notebook position="inside">
-                <page string="Web Favicon" name="favicon">
+                <page string="Web Favicon" name="favicon" groups="web_favicon.group_edit_favicon">
```
Comment (yajo) [inline web_favicon/views/res_company.xml]: If you do what have been asked, this PR would amount to this single line (although fixed), so please squash and leave just 1 commit too.

Thanks!

**Review by yajo — CHANGES_REQUESTED** [review]

File: web_favicon/__manifest__.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
         'web',
     ],
     "data": [
-        "views/res_company.xml",
+        'views/res_company.xml',
```
Comment (yajo) [inline web_favicon/__manifest__.py]: why?

**Review by yajo — CHANGES_REQUESTED** [review]

**Review by yajo — APPROVED** [review]

**Review by felixvillafranca — APPROVED** [review]
Great ¡¡¡

**Comment by tarteo** [comment]
Moved it to https://github.com/OCA/web/pull/645 I was unable to squash @felixvillafranca @Yajo @dreispt

**Comment by yajo** [comment]
Don't worry, I will squash at merge. Let's keep it here if you don't mind.

**Comment by tarteo** [comment]
@Yajo Thanks!

**Review by astirpe — APPROVED** [review]

**Comment by dreispt** [comment]
Build is failing because of the `help_online` module...

**Comment by astirpe** [comment]
The travis issue seems something related with PR #584 on module `help_online`. I think the fixing must be tracked on a different issue, other than this PR.

Can we safely merge this PR then? It only contains a simple change in the view...

**Comment by dreispt** [comment]
If there is no hurry, I would rather wait for the build to be fixed and then rebase this to get a green build.

**Comment by astirpe** [comment]
No hurry, of course. It was just for the sake of satisfaction towards all the contributors of this PR. Lets make the build green first!

**Comment by lmignon** [comment]
@dreispt @astirpe I propose a revert of the merged PR introducing the bug https://github.com/OCA/web/pull/647
For the record I'm not the author of the line introducing the bug.... this line has been added at merge time and squashed into my commits....

**Comment by dreispt** [comment]
@lmignon Stay cool! 🕶 
Code sprints can be fast moving and sometimes people can make wrong calls.
I'm sure it will get fixed soon.

**Comment by yajo** [comment]
https://github.com/OCA/web/pull/647 merged, please rebase

**Comment by tarteo** [comment]
@Yajo Rebase done.

**Comment by astirpe** [comment]
All green! :tada:

**Pull request merged**